### ISSUE TITLE
Not charging to set maximum (MacOS 12 beta)

### ISSUE THREAD

**Issue by leehauser** [davidwernhart/AlDente-Pro-Bug-Tracking#16]

**Comment by leehauser** [comment]
I realize my using the beta makes this a bit of a wildcard...but maybe it's good for you guys to know, and maybe you have some guidance.

I recently updated to the MacOS 12 beta 3. I did not have AlDente set to start at login, and I have MacOS Optimized Charging turned off. When my machine came out of the installation, the battery percentage was 57% (per iStats Menus) and that appeared to be the max. When I started up AlDente I set the max charge to 80%, but the battery does not appear to adding charge to reach the 80% mark; it's just sitting at 57% with the "plugged in" icon.

I haven't yet tried topping up to 100%.

Any thoughts or suggestions on this?
